src/hip_memory.cpp

```diff
@@ -1471,6 +1476,22 @@ hipError_t hipMemcpy3D(const struct hipMemcpy3DParms* p) {
     } else {
         e = hipErrorInvalidValue;
     }
+    return e;
+}
+
+hipError_t hipMemcpy3D(const struct hipMemcpy3DParms* p) {
+    HIP_INIT_SPECIAL_API(hipMemcpy3D, (TRACE_MCMD), p);
+    hipError_t e = hipSuccess;
+    hipStream_t stream = ihipSyncAndResolveStream(hipStreamNull);
+    e = ihipMemcpy3D(p, stream, false);
```

Comment:
Probably better to pass hipStreamNull here and move the resolve back to ihipMemcpy3D